packages/core/src/App/Constants/routes-config.js

```diff
@@ -17,10 +17,8 @@ const Trader = React.lazy(() =>
 );
 
 const Reports = React.lazy(() => {
-    // moduleLoader(() => {
```

Comment:
I  used this function directly for loading the reports in the `packages/reports/src/Constants/routes-config.js` file.